packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableEditor.ts

```diff
@@ -116,17 +116,22 @@ export default class TableEditor {
             let j = 0;
             for (; j < tr.cells.length; j++) {
                 const td = tr.cells[j];
+                const tableRect = normalizeRect(this.table.getBoundingClientRect());
                 const tdRect = normalizeRect(td.getBoundingClientRect());
 
-                if (!tdRect) {
+                if (!tdRect || !tableRect) {
                     continue;
                 }
 
                 const lessThanBottom = y <= tdRect.bottom;
                 const lessThanRight = this.isRTL ? x >= tdRect.right : x <= tdRect.right;
 
                 if (lessThanRight && lessThanBottom) {
-                    if (i == 0 && y <= tdRect.top + INSERTER_HOVER_OFFSET) {
+                    const isOnTableTop = i === 0;
+                    const isOnLeftOrRight = this.isRTL
+                        ? tdRect.right >= tableRect.right && tdRect.right <= tableRect.right + 1
+                        : tdRect.left >= tableRect.left && tdRect.left <= tableRect.left + 1;
+                    if (i == 0 && y <= tdRect.top + INSERTER_HOVER_OFFSET && isOnTableTop) {
```

Comment:
i==0 and isOnTableTop are always the same, right? So we just need to check one of them

Reply:
have you figured out this one?

Reply:
Yes, just changed a = to be i===0